chapter2.adoc

```diff
@@ -29,7 +31,12 @@ the RDSM (except for when explicitly allowed by the RDSM). Privilege levels may
 affect changes in the MTT under purview of the Supervisor Domain Security
 Manager (SDSM) either through an SBI interface into M-mode (or may have the
 ability to edit MTT structures by virtue of how the MTT structure in memory is
-accessible to lower privilege levels).
+accessible to lower privilege levels). Other memory access control mechanisms
+may also be used to enforce supervisor domain isolation for use cases that do
+not require the full flexibility of MTT. For example, (e)PMP configurations
+controlled by M-mode during domain context switching. In this case the MTT-mode
+`Bare` may be used. When MTT mode is `Bare`, MTT is not enforced, but the SDID
```

Comment:
I think we should state:
1. MTT and e(PMP) are always active
2. MTT may be configured to be Bare if granular memory access control is not required
Then we dont need to say anything about SDID etc. MTT is always active - it just happens to be Bare.